### PR TITLE
Upgrade project to yarn berry

### PR DESCRIPTION
**Pull request by puppymati**
The project currently depends on a severely outdated version of yarn. This pr updates it to a more up date version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,23 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+After that install the dependencies with yarn
+
+```bash
+yarn install
+```
+
+Finally run the dev server
 
 ```bash
-npm run dev
```
Comment (Helias) [inline README.md:12]: why did you remove npm reference?

Comment (rosymati) [reply README.md:12]: Unfortunately using npm will not respect the lockfile generated by yarn. The project already depends on yarn anyway so including npm there was an error from the start. Either we use yarn or we just switch the project to vanilla npm. This is pretty much by design when using yarn.

**Review by Lorenzo-Pappalardo — APPROVED** [review]
